main.go

```diff
@@ -192,11 +191,7 @@ func main() {
 			}(endPointName)
 
 		case TypePluginCPFS:
-			go func(endPoint string) {
-				defer wg.Done()
-				driver := cpfs.NewDriver(*nodeID, endPoint)
-				driver.Run()
-			}(endPointName)
+			log.Fatalf("%s is already deprecated, please switch to %s if you are using CPFS 2.0 protocol server", TypePluginCPFS, TypePluginNAS)
```

Comment:
"deprecated" usually means not recommended but still usable. Maybe use word "removed"/"dropped"/"unsupported"?

Reply:
"unsupported" sounds good